test/integration/user-settings.test.js

```diff
@@ -151,6 +151,25 @@ describe('Integration: User settings', async () => {
         });
     });
 
+    test('with prompts muted temporarily doesn\'t see setting showing time remaining', async () => {
+      Date.now = jest.fn(() => new Date(Date.UTC(2018, 4, 4)).valueOf());
+      await slackUser.update({
+        settings: {
+          muteUnfurlPromptsUntil: 1825398771,
```

Comment:
I believe this value is causing the `10 years` in the snapshots.

![image](https://user-images.githubusercontent.com/7718702/46622532-e53b5e80-cb22-11e8-9ba7-d12eabb12c9c.png)

For the snapshot to give us the desired output, i.e. test the new change in this PR, I believe we want this unix timestamp for `muteUnfurlPromptsUntil` to be `before` the `new Date(Date.UTC(2018, 4, 4)` value.

Reply:
> I believe this value is causing the 10 years in the snapshots.

oops 😆 Thanks for clearing this up, I will address this! 👍